### ISSUE TITLE
Escargot AvatarSplitting match error on tip2015/propositional_* problems

### ISSUE THREAD

**Issue by andreasfrom** [gapt/gapt#739]
All five `tip2015/propositional_*` TIP files cause Escargot to throw a `MatchError`:
```scala
scala> val problem = TipSmtImporter.fixupAndLoad( "benchmarks/tip2015/propositional_AndIdempotent.smt2" )
problem: gapt.formats.tip.TipProblem = ...

scala> Escargot.getResolutionProof( problem.toSequent )( problem.ctx.newMutable )
scala.MatchError:  ⊢ ⊥ (of class gapt.proofs.Sequent)
  at gapt.provers.escargot.impl.StandardInferences$AvatarSplitting$.$anonfun$apply$54(inferences.scala:532)
  ...
```

As such, none of the Viper modes work for these files.

**Comment by jannikvierling** [comment]
This happens because ⊥ should not appear as an atom of a clause. I will check how the ⊥ got there.

**Comment by jannikvierling** [comment]
I have pushed a potential fix (see [33e242e49](https://github.com/gapt/gapt/commit/33e242e497c3f1bdde74a3894b5f127661563e2a)). Let me know if this does not work for you.

**Comment by andreasfrom** [comment]
It prevents me from solving isaplanner/prop_02 but I don't have time to look into the details right now.
